installer/generate_secrets.py

```diff
@@ -221,6 +221,11 @@ def _gafaelfawr(self):
                 f"Invalid gafaelfawr cloudsql value {use_cloudsql}"
             )
 
+        self.input_field("gafaelfawr", "ldap", "Use LDAP? (y/n):")
```

Comment:
Should the use-cilogon-or-github go in the else clause, or do we still need it even if we're using LDAP?

Reply:
They're independent; you can use any authentication source and then still use LDAP for data (although in practice we will probably only use LDAP for the CILogon and OpenID Connect cases).